frontend/src/App.jsx

```diff
@@ -51,30 +48,24 @@ function App() {
       <ScrollToTop />
       {user && <Announcement />}
       {!loading && (
-        <Switch>
-          {user && [
-            <Route key="appointments" exact path="/psychologue/mes-seances" component={Appointments} />,
-            <Route key="new-appointments" exact path="/psychologue/nouvelle-seance" component={NewAppointment} />,
-            <Route key="patients" exact path="/psychologue/mes-patients" component={Patients} />,
-            <Route key="new-patient" exact path="/psychologue/nouveau-patient" component={AddEditPatient} />,
-            <Route key="edit-patient" exact path="/psychologue/modifier-patient/:patientId" component={AddEditPatient} />,
-            <Route key="reimbursment" exact path="/psychologue/mes-remboursements" component={Reimbursement} />,
-            <Route key="psy-profil" path="/psychologue/mon-profil" component={PsyProfile} />,
-          ]}
-          <Route exact path="/psychologue/login/:token?" component={Login} />
-          <Route exact path="/trouver-un-psychologue" component={FindPsychologist} />
-          <Route exact path="/mentions-legales" component={LegalNotice} />
-          <Route exact path="/donnees-personnelles-et-gestion-des-cookies" component={PersonalData} />
-          <Route exact path="/faq" component={Faq} />
-          <Route exact path="/stats" component={Statistics} />
-          <Route exact path="/" component={Landing} />
-          <Route path="/psychologue/">
-            <Redirect to={user ? '/psychologue/mes-seances' : '/psychologue/login'} />
-          </Route>
-          <Route path="/">
-            <Redirect to={user ? '/psychologue/mes-seances' : '/'} />
-          </Route>
-        </Switch>
+        <React.Suspense fallback={<></>}>
+          <Switch>
+            <Route exact path="/psychologue/login/:token?" component={Login} />
+            {user && <Route path="/psychologue/" component={PsychologistRouter} />}
+            <Route exact path="/trouver-un-psychologue" component={FindPsychologist} />
+            <Route exact path="/mentions-legales" component={LegalNotice} />
+            <Route exact path="/donnees-personnelles-et-gestion-des-cookies" component={PersonalData} />
+            <Route exact path="/faq" component={Faq} />
+            <Route exact path="/stats" component={Statistics} />
+            <Route exact path="/" component={Landing} />
+            <Route path="/psychologue/">
+              <Redirect to="/psychologue/login" />
+            </Route>
```

Comment:
Can this route be in conflict with the one from PsychologistRouter (which redirects to "mes-seances")?

When logged in, `/psychologue/login` redirects to `/psychologue/mes-seances`, but I couldn't see if it was doing: 

- `/psychologue/` -> `/psychologue/login` -> `/psychologue/mes-seances` 
- or directly `/psychologue/` -> `/psychologue/mes-seances`.